packages/ui/src/components/molecules/FormInput/FormInput.tsx

```diff
@@ -98,6 +100,7 @@ const FormInput = ({
           onEndFocus={handleEndFocus}
           numberOfLines={null}
           readOnly={readOnly}
+          multiline={multiline}
```

Comment:
Display should be multiline too when this props is enabled (should be modify in input component  I think).
Currently, you can write several lines but only the last one is displayed, that's not user friendly.